### DISCUSSION TITLE
Ability has Procced

### DISCUSSION THREAD

**Discussion by mikemangu** [Hekili/hekili#768] [Q&A]
Hey, trying to figure out how to set up an ability to be recommended when a specific one has proc,

in this case Death Strike, I only want to use it when it has proc. buff.death_strike.up?

**Comment by Hekili** [comment]
Hi, due to time constraints, I don't help people write custom priorities.  The best you can do is test things in SimulationCraft and refer to their wiki for general syntax and so forth.

**Reply by mikemangu** [reply]
Is there a syntax guide for SimulationCraft? Havent been able to find one.
